packages/editor-ui/src/init.ts

```diff
@@ -63,5 +60,17 @@ export async function initializeAuthenticatedFeatures() {
 		await nodeTypesStore.getNodeTranslationHeaders();
 	}
 
+	if (settingsStore.isCloudDeployment) {
+		const results = await Promise.allSettled([cloudPlanStore.initialize(), initializeCloudHooks()]);
+		results.forEach((result, index) => {
+			if (result.status === 'rejected') {
+				console.error(
+					`Failed to initialize ${index === 0 ? 'cloud plan store' : 'cloud hooks'}:`,
+					result.reason,
+				);
+			}
+		});
```

Comment:
The `initializeCloudHooks` call does not require the user to be authenticated and should happen in the `initializeCore` block so we can actually run the `app.mounted` hook as well.

Reply:
Ah, yes. Moved.